docs/source/en/_toctree.yml

```diff
@@ -44,7 +44,8 @@
       title: Use tokenizers from 🤗 Tokenizers
     - local: multilingual
       title: Inference for multilingual models
-    - sections:
+    - isExpanded: false
```

Comment:
Normally this shouldn't be touched by this PR

Reply:
for some reason, these are automatically updated after make fixup, reverting them back